specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2023-04-15-preview/cluster.json

```diff
@@ -1237,6 +1237,18 @@
       "type": "object",
       "description": "The network properties.",
       "properties": {
+        "outboundDependenciesManagedType": {
```

Comment:
>outboundDependenciesManagedType [](http://example.com/codeflow?start=9&length=31)

I feel like I'm missing some context for this documentation to make sense. What kind of outbound dependencies is this talking about, and what kind of managing is done to them? Anthing you could add to clarify?

Reply:
Let me add one document here to explain the change. [HDInsight Cluster Spec Change Proposal] https://microsoftapc-my.sharepoint.com/:w:/g/personal/xinli1_microsoft_com/EaQq56jg_WJCr1-jlrU1hRQBVPl09ltZ5sCI6hdS-S9AfQ?e=IZ1zrJ

HDInsight provides the experience of Hadoop cluster as a service. Internally they are a set of VMs and their virtual network configurations. Previous, we let the customer bring & manage their own network solutions including NSG, Firewall, and 3rd party integrations while today some Cx require us to manage. That's how we define 'Managed' and 'External'